### PR TITLE
Correct Chinese and Japanese translations

### PR DESCRIPTION
**Pull request by Xujiayao**
Thank you for making such a fantastic Swing Look and Feel! I corrected some Chinese and Japanese translation errors. 

Btw, I cannot find the Chinese and Japanese translation files of `theme_settings.properties`. Is it to copy the original files directly and rename the new ones to `_zh` and `_ja`? If so, please tell me, don't merge the pull request for now, I will translate them too.

Thank you!

### CODE REVIEW

**Comment by weisJ** [comment]
Thanks for fixing the translations.

> Btw, I cannot find the Chinese and Japanese translation files of theme_settings.properties. Is it to copy the original files directly and rename the new ones to _zh and _ja?

That is correct :)

**Comment by Xujiayao** [comment]
Added Chinese and Japanese translations for `theme_settings.properties`. It's ready to merge now :D

File: core/src/main/resources/com/github/weisj/darklaf/task/darklaf_ja.properties [hunk 1]
```diff
@@ -24,16 +24,16 @@
...
 Actions.minimize                   = 最小化
-Actions.restore                    = 戻す
-Actions.closableTabbedPane.close   = 閉じる。Alt-クリックして他を閉じる
+Actions.restore                    = 元のサイズに戻す
+Actions.closableTabbedPane.close   = 閉じる。Alt キーを押しながらクリックすると、この夕ブ以外の他の夕ブを閉じます。
 Actions.save                       = 保存する
 Actions.revert                     = 元に戻す
 Actions.predefinedValues           = 事前定義された値
 
-Labels.colorChooser                = カラーチューザー
```
Comment (weisJ) [inline core/src/main/resources/com/github/weisj/darklaf/task/darklaf_ja.properties:37]: Please squash those changes into the first commit :)
Same applies for the changes in `darklaf_zh` and `tabFrame_ja`

File: core/src/main/resources/com/github/weisj/darklaf/task/tabFrame_ja.properties [hunk 2]
```diff
@@ -22,11 +22,11 @@
...
 # suppress inspection "UnusedProperty" for whole file
 popup.moveTo            = 移動
-popup.moveTo.north      = 左上隅
-popup.moveTo.south      = 右下隅
-popup.moveTo.east       = 右上隅
-popup.moveTo.west       = 左下隅
-popup.moveTo.north_east = 右上隅
-popup.moveTo.north_west = 左上隅
-popup.moveTo.south_east = 右下隅
-popup.moveTo.south_west = 左下隅
```
Comment (weisJ) [inline core/src/main/resources/com/github/weisj/darklaf/task/tabFrame_ja.properties]: Please squash this commit into the previous one :)

**Review by weisJ — CHANGES_REQUESTED** [review]

**Comment by Xujiayao** [comment]
Sorry for being unfamiliar with Pull requests and Git squash, it's now a single commit. 😂

**Comment by weisJ** [comment]
No worries :) Thanks a lot for the translations

**Pull request merged**